### ISSUE TITLE
GhostNet中的线性变换操作

### ISSUE THREAD

**Issue by PlasticBagBand** [huawei-noah/Efficient-AI-Backbones#209]
你好，我在阅读ghostnet的源码的时候发现在GhostModule中的cheap_operations中，发现论文中的“线性变换”应该就是分组卷积。但是”线性变换“中为什么会有ReLU层呢？谢谢！
` self.cheap_operation = nn.Sequential(` 
`           nn.Conv2d(init_channels, new_channels, dw_size, 1, dw_size//2, groups=init_channels, bias=False),` 
`            nn.BatchNorm2d(new_channels),` 
`            nn.ReLU(inplace=True) if relu else nn.Sequential(), # ？？？` 
`        )`

**Comment by iamhankai** [comment]
线性变换单指分组卷积，relu层是整个特征拼接起来后的激活函数

**Comment by PlasticBagBand** [comment]
好的，我懂了，谢谢你！！！
